controllers/api/auth_test.go

```diff
@@ -0,0 +1,283 @@
+package api_controllers_test
+
+import (
+	. "go-google-scraper-challenge/helpers/test"
+	"go-google-scraper-challenge/initializers"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+type LoginResponse struct {
+	AccessToken  string `json:"access_token"`
+	ExpiresIn    int    `json:"expires_in"`
+	RefreshToken string `json:"refresh_token"`
+	TokenType    string `json:"token_type"`
+}
+
+type ErrorResponse struct {
+	Error            string `json:"error"`
+	ErrorDescription string `json:"error_description"`
+}
+
+var _ = Describe("AuthController", func() {
+	Describe("POST /login", func() {
+		Context("given valid params", func() {
+			It("responses with status ok", func() {
+				FabricateUser("dev@nimblehq.co", "password")
+				oauth_client := FabricateOAuthClient()
+				body := RequestBody(map[string]string{
+					"client_id":     oauth_client.ClientID,
+					"client_secret": oauth_client.ClientSecret,
+					"username":      "dev@nimblehq.co",
+					"password":      "password",
+					"grant_type":    "password",
+				})
+				response := MakeRequest("POST", "/api/v1/login", body)
+
+				Expect(response.StatusCode).To(Equal(http.StatusOK))
+			})
+
+			It("response with token data", func() {
```

Comment:
Avoid using `data` as it's quite inaccurate:

```suggestion
			It("response with a valid token", func() {
```

Reply:
@olivierobert fixed on [160468b](https://github.com/carryall/go-google-scraper-challenge/pull/26/commits/160468bf6b37d33cf528f0a8664be54b4c07c1c7)